lib/models/case.js

```diff
@@ -64,7 +63,25 @@ class Case {
   }
 
   static list() {
-    return this.Model.query().select();
+    return this.query().select().whereNot({ status: 'autoresolved' });
```

Comment:
This is probably too much of our implementation leaking into the framework.

I'd probably go for something like:

```
{
  include: ['new', 'active'],
  exclude: ['resolved']
}
```

So that we can either whitelist or blacklist certain statuses by passing them in the query.

Reply:
Those params would probably be mutually exclusive in reality - you'd either provide a list of statuses to include, or to exclude, but it would be unlikely to be both.